dash-info.yaml

```diff
@@ -1,10 +1,2479 @@
-pkg_help_description: >
+pkg_help_description: >-
```

Comment:
Is this file generated?

Reply:
@Marc-Andre-Rivet This file is user-authored, and not auto-generated.